frontend/src/store/trashCountStore.ts

```diff
@@ -0,0 +1,26 @@
+import { create } from "zustand";
+import * as NotesApi from "../network/notes_api";
+
+async function fetchTrashCount() {
+  return (await NotesApi.fetcDeletedNotes()).length;
+}
+
+interface TrashCountState {
+  count: number;
+  increment: () => void;
+  decrement: () => void;
```

Comment:
I'd rename these to `incrementTrashCount` and `decrementTrashCount` bc we use these in the code with no context and what they mean is not super clear at the frist glance. 

Also can't we use just increment with -1?